feat[gen1][core] ENG-7238 remove query API and apiEndpoint field (#3681)
BuilderIO/builder: .changeset/pink-fans-serve.md

---
"@builder.io/sdk": major
"@builder.io/react": major
---

> Permanently removes `apiEndpoint` prop from `builder.get()` and `builder.getAll()` which would have options `'content'` and `'query'`. Content API will be the default API endpoint is used for the content fetching.

Reviewer: ```suggestion
Permanently removes the `apiEndpoint` prop from `builder.get()` and `builder.getAll()` which had options `'content'` and `'query'`. Content API is now the only possible API endpoint for content fetching.
```
